skimage/util/arraycrop.py

```diff
@@ -4,138 +4,18 @@
 """
 
 import numpy as np
-from numpy.lib.arraypad import _validate_lengths
 
+# After numpy 1.15, a new backward compatible function have been implemented.
+# See https://github.com/numpy/numpy/pull/11966
+old_numpy = False
+try:
+    from numpy.lib.arraypad import _as_pairs
+except ImportError:
+    from numpy.lib.arraypad import _validate_lengths
+    old_numpy = True
```

Comment:
@sciunto, I prefer to have the NumPy version check explicit here:

```
from distutils import LooseVersion as Version
old_numpy = Version(np.__version__) < Version('1.16')
if old_numpy:
    from numpy.lib.arraypad import _validate_lengths
else:
    from numpy.lib.arraypad import _as_pairs
```
This makes it clearer (though code) whether 1.15 is inclusive or exclusive.

Reply:
I made this change :)

Reply:
it's distutils.version import... ;)

Reply:
Oops, sorry! =\